ONSdigital/dp-image-api
# Feature/include collections in healthcheck

### What

- Update mongodb library so that collections that be passed into the healthcheck client
- Update the dp-net library, and make code changes to reflect the fact that getKeyId has changed package

### How to review

- Check that if the collection is not in the database the healthcheck fails and if it is the healthcheck passes
- Check that happy with the dp-net changes

### Who can review

Anyone
